### PR TITLE
chore: add style standards check

### LINKED ISSUE #87: Require conformance to style standards
#### Issue Summary

In the `CONTRIBUTING.md` file there is a section on Style Guidelines & Naming Conventions. Please review and implement a check in Travis CI. If there are any style errors, either fix or create a rule to exclude the error.

Before writing any code, we suggest you review our `CONTRIBUTING.md` file and propose your solution for discussion here in the comments. Thank you!

#### Aceptance Criteria

* TravisCI checks that the code conforms to our Style Guidelines & Naming Conventions as defined in our `CONTRIBUTING.md` file
* tests pass locally, in Travis CI and using Docker

### PR DESCRIPTION
**Pull request by pangaunn**
# Fixes # 
Fixes #87 
### Checklist
- [x] I have made a material change to the repo (functionality, testing, spelling, grammar)
- [x] I have read the [Contribution Guide] and my PR follows them.
- [x] I updated my branch with the master branch.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation about the functionality in the appropriate .md file
- [ ] I have added in line documentation to the code I modified

### Short description of what this PR does:
- Add short description in each function in header and add blank line between @param and @return.
- Add travis to run phpcs.
- Change toArray to public. This function is in the USAGE.md file I think it should be public.

If you have questions, please send an email to [Sendgrid](mailto:dx@sendgrid.com), or file a Github Issue in this repository.

### CODE REVIEW

**Comment by SendGridDX** [comment]
[CLA assistant check](https://cla.sendgrid.com/sendgrid/smtpapi-php?pullRequest=90) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.sendgrid.com/sendgrid/smtpapi-php?pullRequest=90) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.sendgrid.com/check/sendgrid/smtpapi-php?pullRequest=90) it.</sub>

File: .travis.yml [hunk 1]
```diff
@@ -3,6 +3,7 @@ before_script:
 - composer install --dev --no-interaction
 - cd test
 script:
+- "php ../vendor/bin/phpcs -n --exclude=PEAR.Commenting.FileComment,PEAR.Commenting.ClassComment,PEAR.Commenting.FunctionComment ../lib/"
```
Comment (peter279k) [inline .travis.yml]: How about using the following `phpcs` command to check the `PSR-2` coding style?

```php
./vendor/bin/phpcs -n --standard=PSR2 lib/ test/
```

I think this command I suggest is good enough to use :).

Comment (pangaunn) [reply .travis.yml]: Thank you for review but In [CONTRIBUTING.md](https://github.com/sendgrid/smtpapi-php/blob/master/CONTRIBUTING.md#style-guidelines-and-naming-conventions) it's using PEAR standard. I will add /test in script.

Comment (peter279k) [reply .travis.yml]: Mm....The coding standard is different from [sendgrid-php](https://github.com/sendgrid/sendgrid-php/blob/master/CONTRIBUTING.md). I think it also uses the PSR-2 coding style.

If this repo uses `PEAR` coding style, this comment I write will not be concerned :).

**Comment by thinkingserious** [comment]
Hello @pangaunn,

Apologies for the delayed code review. 

I attempted to resolve the conflicts, but I do not have permission to push to your branch. Would you mind resolving the conflicts or granting me permission to push to your branch?

Thanks!

Elmer

**Comment by pangaunn** [comment]
Hello @thinkingserious I just send you an invite please check.

**Review by thinkingserious — APPROVED** [review]

**Pull request merged**